spec/test.js

```diff
@@ -0,0 +1,13 @@
+// Require the built in 'assertion' library
+var assert = require('assert');
```

Comment:
My suggestion here is to use `const` since assert is not being reassigned anywhere